core/storage/classifier/gae_models_test.py

```diff
@@ -111,6 +111,115 @@ def test_query_new_and_pending_training_jobs(self):
             feconf.TRAINING_JOB_STATUS_PENDING)
         self.assertEqual(offset, 2)
 
+    def test_query_new_and_pending_training_jobs_with_multiple_fetches(self):
+        offset = 0
+        next_scheduled_check_time = datetime.datetime.utcnow()
+        # Creating 14 jobs out of which 12 will be fetched in steps.
+        classifier_models.ClassifierTrainingJobModel.create(
```

Comment:
Rather than creating 10 models, I suggest swapping out the constant to something lower (e.g. 2) to more easily simulate the limit being hit. This also avoids the test being brittle if the constant ever changes in the future (this test is just verifying that a limit is enforced, not the specific limit itself).

Reply:
But don't you think that changing 10 to 2 in gae_models will change its basic functionality?

Reply:
I think I misunderstood you. Have made the changes. PTAL.

Reply:
No, I think the point @BenHenning is making (and which I would also make) is use self.swap() to swap out the value of NEW_AND_PENDING_TRAINING_JOBS_FETCH_LIMIT to something like 2, so that you can check offsetting more easily.

Reply:
Done. PTAL